recipes/doxygen/all/conanfile.py

```diff
@@ -10,7 +10,15 @@ class DoxygenConan(ConanFile):
     homepage = "https://github.com/doxygen/doxygen"
     license = "GPL-2.0-or-later"
     url = "https://github.com/conan-io/conan-center-index"
-    settings = "os", "arch", "compiler", "build_type"
+    settings = "os", "os_build", "arch", "arch_build", "compiler", "build_type"
```

Comment:
This is what we have documented in our FAQ, there rest I can not comment on as I don't know them well enough

```suggestion
    settings = "os", "os_build", "arch", "arch_build", "build_type"
```